### PR TITLE
Register ahmed-raza.is-a.dev

### PR DESCRIPTION
**Pull request by Webdev-Ahmed**
Register ahmed-raza.is-a.dev with CNAME record pointing to ajgkc7rd.up.railway.app.

### CODE REVIEW

File: domains/ahmed-raza.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Webdev-Ahmed",
+           "email": "",
+           "discord": "876045368033439754",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.qZhlFuXJ00kRuu5CQBkkXtKCMgk0WdF3Oe6LyROt81S6d_ZnV18n61aPM9iq-GFjcuj3F8Gi-xSAQomuxNuPovdlKV6uaTRuub3hmZjqgRd3WXszLiyH7KeTU4DTUKAHFGb9t1lA1BluU6Rmqz_Y086nwHif2KmwPIzECjiPVKtxgan7B7Tb7vRaheAvH6BzwBKM611EF1qPw5GrSoGyhmeHGDeLesuRM5kc4-jKC-Lz8DcU_zU8yb4qY551119WdI-PtqX8esO7zNe9nUJ2PhTaTuSUqEJ128Www8q6aMkwdehX5lgvtn7ikSatBZ40T-d8PK9MV3dAt0V3_KvjYA.tMapL6OZraMRXwsbVvz_Zw.pJGrq5NEZG4VrhJqMmYscE6g9KnhpMgjx8blCJ6XjAKFDc9PkwmVqc22c-A79KhOIbCNK4F1u8NXIv7mc1T4wqXLsMIh5LyHvHCXCzEo2Gs.q2DAYmtBvfYaZQz23UTR0g"
+        },
+    
+        "record": {
+            "CNAME": "ajgkc7rd.up.railway.app"
```
Comment (Stef-00012) [inline domains/ahmed-raza.json:10]: I cannot reach your website, railway says "Nothing here... yet"

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/ahmed-raza.json (hunk 1 above)
Comment (Webdev-Ahmed) [reply domains/ahmed-raza.json:10]: Can i use netlify or vercel, i cant ger railway to work

Comment (orxngc) [reply domains/ahmed-raza.json:10]: > Can i use netlify or vercel, i cant ger railway to work

you can use netlify but not vercel @Webdev-Ahmed

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**